src/daemon/mod.rs

```diff
@@ -443,8 +440,8 @@ pub(super) async fn start(
                     crate::rpc::f3::get_f3_rpc_endpoint().to_string(),
                     initial_power_table,
                     finality,
-                    std::env::var("FOREST_F3_DB_PATH")
-                        .unwrap_or(default_f3_db_path.display().to_string()),
+                    std::env::var("FOREST_F3_ROOT")
+                        .unwrap_or(default_f3_root.display().to_string()),
```

Comment:
Is there any documentation we should update? If not, is there any documentation we should create? :D

Reply:
Will make a follow-up PR in the docs repo